Add asm.js feature.

kanaka opened the pull request.

cvrebert commented on features-json/asmjs.json:
```diff
@@ -0,0 +1,241 @@
+      "9.9":"n"
+    }
+  },
+  "notes":"Sample notes for feature, explain partial support here",
```
Remove sample text

dstorey commented on features-json/asmjs.json:
```diff
@@ -0,0 +1,241 @@
+    },
+    {
+      "url":"https://dev.modern.ie/platform/changelog/10532-pc/",
+      "title":"MS Edge support announcement"
```
Should really be Microsoft Edge rather than MS Edge (former is correct name, latter is short hand for places like Twitter)

dstorey commented on features-json/asmjs.json:
```diff
@@ -0,0 +1,241 @@
+      "title":"Source for spec and tools"
+    },
+    {
+      "url":"https://dev.modern.ie/platform/changelog/10532-pc/",
```
This is probably a better link http://blogs.windows.com/msedgedev/2015/05/07/bringing-asm-js-to-chakra-microsoft-edge/

dstorey commented on features-json/asmjs.json:
```diff
@@ -0,0 +1,241 @@
+  "notes":"Sample notes for feature, explain partial support here",
+  "notes_by_num":{
+    "1":"Chrome does not support Ahead-Of-Time compilation but performance doubled in Chrome 28: https://en.wikipedia.org/wiki/Asm.js#Implementations",
+    "2":"MS Edge build 10532+ enables unprefixed support for asm.js"
```
This isn't really true. It was behind a flag in EdgeHTML 12, and enabled by default in the first preview of what will be EdgeHTML 13 (which will likely be labelled Edge 2 on CanIUse…) once the upcoming Microsoft Edge version is included here.

Fyrd commented:
@kanaka Can you address these comments? Thanks.

kanaka commented:
@Fyrd I've addressed the comments. I marked Edge 1 as "n" but kept the comment since that is useful information to know until Edge 2 is added to the list.

Fyrd commented:
Great, thanks!
